.github/workflows/release.yml

```diff
@@ -33,9 +33,22 @@ jobs:
       - name: Install Dependencies
         run: pnpm install --ignore-scripts
 
-      - name: Create Release Pull Request or Publish to npm
+      - name: Create Release Pull Request
         uses: changesets/action@v1
         with:
-          publish: pnpm release
+          version: pnpm changeset version
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Update pnpm-lock.yaml
```

Comment:
Okay, I agree this is needed. My only note is that changesets won't bump versions of the dependencies in the `app/reaction` unless we are doing a major release because we are using ^ for all plugins and in turn, the lock file won't change.